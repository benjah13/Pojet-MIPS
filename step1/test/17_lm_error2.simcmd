# valeur a inserer de la memoire trop grande

lm 60 10
lm 60 ffa
