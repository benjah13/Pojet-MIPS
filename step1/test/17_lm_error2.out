The value 10 has been loaded in the address 0x60
Error : The 2nd argument of lm isn't a valide value
